geospatial/src/CMakeLists.txt

```diff
@@ -1,24 +1,27 @@
-gz_get_libsources_and_unittests(sources gtest_sources)
+if (TARGET ${PROJECT_LIBRARY_TARGET_NAME}-graphics)
+  gz_get_libsources_and_unittests(sources gtest_sources)
+  gz_add_component(geospatial
+    SOURCES ${sources}
+    DEPENDS_ON_COMPONENTS graphics
+    GET_TARGET_NAME geospatial_target)
 
-gz_add_component(geospatial
-  SOURCES ${sources}
-  DEPENDS_ON_COMPONENTS graphics
```

Comment:
I thought the `DEPENDS_ON_COMPONENTS` parameter would generate a complaint if the needed component wasn't available, but apparently it doesn't

Reply:
That was also my understanding, but does not seem to be the case.